docs/app/pages/components/components-sections/timeline/timeline/timeline.component.html

```diff
@@ -37,3 +37,13 @@
     </tr>
 </uxd-api-properties>
 
+<p>The following code can be used to create the example above:</p>
+
+<tabset>
+  <tab heading="HTML">
+    <uxd-snippet [content]="snippets.compiled.appHtml" language="html"></uxd-snippet>
+  </tab>
+  <tab heading="JavaScript">
```

Comment:
Change heading to TypeScript. We can also remove the language attributes here. These are no longer needed